### ISSUE TITLE
Crashes with Latin ə character

### ISSUE THREAD

**Issue by zsbee** [wibosco/GhostTypewriter#15]
```
 *** Terminating app due to uncaught exception 'NSRangeException', reason: 'NSMutableRLEArray objectAtIndex:effectiveRange:: Out of bounds'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff23c4f02e __exceptionPreprocess + 350
	1   libobjc.A.dylib                     0x00007fff50b97b20 objc_exception_throw + 48
	2   Foundation                          0x00007fff25805cd9 blockForLocation + 0
	3   Foundation                          0x00007fff2580406b -[NSConcreteAttributedString attribute:atIndex:effectiveRange:] + 58
```

**Comment by wibosco** [comment]
@zsbee I'm having some trouble reproducing this crash. 

I've added a `ə` character to the example project's text but without causing it to crash. 

Might be related to OS version - what version are you seeing the crash on?

I did notice that having a `ə` character in the text caused any other applied attributes after the `ə` to be applied to the wrong characters. Are you setting the `attributedText` property on the label that crashes? If so could you share that `NSAttributedString`?

**Comment by zsbee** [comment]
of course :)

`label.numberOfLines = 0
        label.typingTimeInterval = 0.05
        let attributedString = NSAttributedString(string: "/pər\'spect/ \nThe art of exploring multiple internal and external viewpoints to gain clarity and insight and drive action.", attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: 21, weight: .light)])
        label.attributedText = attributedString`

When I change the ə to e it works

**Comment by wibosco** [comment]
@zsbee thanks, I was able to reproduce the crash.

I've released a new version (2.0.1) that should resolve the bug. 

Basically the string index that I store to determine the next character to reveal went out of sync with the attributedstring's indexes.

**Comment by zsbee** [comment]
Thanks man! Will give it a try now!
